src/util/tests.json

```diff
@@ -947,7 +947,7 @@
     }],
     "streamable": [{
         "name": "regular video",
-        "url": "https://streamable.com/03r3c2",
+        "url": "https://streamable.com/p7blpp",
```

Comment:
it might be a better idea to use some older streamable video that is still up, because they purge videos after some time

Reply:
2 months isn't old enough for you?

Reply:
no because they purge videos after 90 days

Reply:
ok, i'll update the link in a sec